src/OFS/Image.py

```diff
@@ -520,6 +520,9 @@ def _read_data(self, file):
         n = 1 << 16
 
         if isinstance(file, str):
+            raise ValueError("Must be bytes")
```

Comment:
When I look at other packages which share the same code for python2/3 they have something like this.
http://pythonhosted.org/six/#six.binary_type
http://pythonhosted.org/six/#six.text_type
For example flask has its own _compat.py  to avoid the dependency to six. https://github.com/pallets/flask/blob/master/flask/_compat.py

I think to port the Zope2 codebase something like this is needed as well.